### DISCUSSION TITLE
Bundle doesn't break out when applying fixedPricePerUnit

### DISCUSSION THREAD

**Discussion by paragopinath** [Shopify/function-examples#379] [Feedback: Cart Transform API]
we are testing out the new dev preview of 'Cart Transform Function API - Fixed Price Preview' where we are trying to pass the `fixedPricePerUnit` price adjustment, but we noticed the bundle breakout is not working as expected and there is no error in the function log as well.

Function API Version: `unstable`

Here is the function output
<img width="933" alt="Screen Shot 2023-10-20 at 11 32 55 AM" src="https://github.com/Shopify/function-examples/assets/56590166/0a792f09-2216-45f3-a3e9-536b1de0a89f">

**Comment by dmasciotra** [comment]
Just to confirm:
Are you using a development store with the [Cart Transform Dev Preview](https://shopify.dev/docs/api/release-notes/developer-previews#cart-transform-developer-preview)?

**Reply by paragopinath** [reply]
No, I'm using a dev store that has checkout extensibility dev preview, Now I see a new dev preview option to select cart transform during the dev store creation, I'll test and confirm.

**Comment by paragopinath** [comment]
confirming it is working as expected when we use store with cart transform preview enabled!
